src/ui/utils/timeline.js

```diff
@@ -1,4 +1,4 @@
-const timelineMarkerWidth = 7;
+const timelineMarkerWidth = 11;
```

Comment:
This number is repeated in so many places... Do we have nowhere to export this kind of thing from so it is shared across wherever it is needed?

I feel like any changes here could make it pretty easy to get out-of-sync.

Reply:
I would recommend adding a constants.js file in ui/ if it makes sense.

Reply:
Good call - went ahead with creating a constants. Thanks for the suggestions!